### PR TITLE
0.2.0 README update

### PR DESCRIPTION
**Pull request by thundergolfer**
I know of more changes that should be made to README, but I want to keep this minimal and concerned only with the recent release.

----

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] Does not include precompiled binaries, eg. `.par` files. See [CONTRIBUTING.md](https://github.com/bazelbuild/rules_python/blob/master/CONTRIBUTING.md) for info
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature (please, look at the "Scope of the project" section in the README.md file)
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] Other... Please describe:

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,6 +5,8 @@
 
 ## Recent updates
 
+* 2021-03-31: Release [`0.2.0` was published](https://github.com/bazelbuild/rules_python/releases/tag/0.2.0).
```
Comment (thundergolfer) [inline README.md:8]: Open to suggestions on what to include as detail here.

**Comment by thundergolfer** [comment]
Will ping philwo and announce release once this is merged and we're all happy.

File: README.md (hunk 1 above)
Comment (alexeagle) [reply README.md:8]: IMO we should just remove the "Recent updates" section as it's totally redundant with the releases page. We could migrate the other content here to the Release it belongs on, or maybe leave here with "Releases prior to 2021 have their notes below"

File: README.md [hunk 2]
```diff
@@ -61,8 +63,8 @@ To import rules_python in your project, you first need to add it to your
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 http_archive(
     name = "rules_python",
-    url = "https://github.com/bazelbuild/rules_python/releases/download/0.1.0/rules_python-0.1.0.tar.gz",
-    sha256 = "b6d46438523a3ec0f3cead544190ee13223a52f6a6765a29eae7b7cc24cc83a0",
+    url = "https://github.com/bazelbuild/rules_python/releases/download/0.2.0/rules_python-0.2.0.tar.gz",
+    sha256 = "778197e26c5fbeb07ac2a2c5ae405b30f6cb7ad1f5510ea6fdac03bded96cc6f",
```
Comment (alexeagle) [inline README.md:67]: future: maybe we want the release script to do this? on rules_nodejs we do the bumps here as part of `npm version` - dunno if python bumpversion library would be useful

**Review by alexeagle — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (thundergolfer) [reply README.md:8]: Agreed. I'll remove it.

**Comment by thundergolfer** [comment]
@philwo could we have `0.2.0` populated in the cache? Link --> https://github.com/bazelbuild/rules_python/releases/tag/0.2.0

**Comment by philwo** [comment]
@thundergolfer Done! :)

```
$ mirror.sh https://github.com/bazelbuild/rules_python/releases/download/0.2.0/rules_python-0.2.0.tar.gz
Here's your snippet for the WORKSPACE file:

load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")

http_archive(
    name = "rules_python",
    sha256 = "778197e26c5fbeb07ac2a2c5ae405b30f6cb7ad1f5510ea6fdac03bded96cc6f",
    urls = [
        "https://mirror.bazel.build/github.com/bazelbuild/rules_python/releases/download/0.2.0/rules_python-0.2.0.tar.gz",
        "https://github.com/bazelbuild/rules_python/releases/download/0.2.0/rules_python-0.2.0.tar.gz",
    ],
)
```

**Pull request merged**